documentation/todo-app/src/components/todoMain.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import TodoItems from './todoItems';
+import './todo_main.css'
+
+export default class TodoMain extends Component {
+    render() {
+        const todos = [
+            {
+                'description': 'Get out of bed',
+                'deadline': 'Wed Sep 13 2017'
+            },
+            {
+                'description': 'Brush teeth',
+                'deadline': 'Thu Sep 14 2017'
+            },
+            {
+                'description': 'Eat breakfast',
+                'deadline': 'Fri Sep 15 2017'
+            }
+        ]
+
+        const main_todos = todos.map((todo, i) => (
+            <TodoItems description = { todo.description } deadline = { todo.deadline } />
```

Comment:
First, we don't normally put spaces around the `=`. Second, if you run your app and then open the dev console in whatever browser you are using, you'll see this warning:

> Warning: Each child in an array or iterator should have a unique "key" prop.

Did you cover this with Nasir? When we render React JSX in any kind of loop, we have to specifically tell React what each element is... basically we have to give items IDs. You can check the [docs here](https://reactjs.org/docs/lists-and-keys.html#keys) for more info.